src/agent/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 
 find_package(Qt5 5.15 REQUIRED COMPONENTS Core Network)
+find_library(WBEMUUID_LIBRARY wbemuuid REQUIRED)
```

Comment:
This one should be required and used only on windows

Reply:
Fixed indirectly with different commit